gbpservice/nfp/config_orchestrator/agent/loadbalancerv2.py

```diff
@@ -0,0 +1,457 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import ast
+import copy
+
+from gbpservice.nfp.common import constants as const
+from gbpservice.nfp.config_orchestrator.agent import common
+from gbpservice.nfp.config_orchestrator.agent import topics as a_topics
+from gbpservice.nfp.core import common as nfp_common
+from gbpservice.nfp.lib import transport
+
+from neutron_lbaas.db.loadbalancer import loadbalancer_dbv2
+
+from oslo_log import helpers as log_helpers
+from oslo_log import log as oslo_logging
+import oslo_messaging as messaging
+
+LOGGER = oslo_logging.getLogger(__name__)
+LOG = nfp_common.log
+
+"""
+RPC handler for Loadbalancer service
+"""
+
+
+class Lbv2Agent(loadbalancer_dbv2.LoadBalancerPluginDbv2):
+    RPC_API_VERSION = '1.0'
+    target = messaging.Target(version=RPC_API_VERSION)
+
+    def __init__(self, conf, sc):
+        super(Lbv2Agent, self).__init__()
+        self._conf = conf
+        self._sc = sc
+        self._db_inst = super(Lbv2Agent, self)
+
+    def _filter_service_info_with_resource(self, lb_db, core_db):
+        updated_db = {'subnets': [],
+                      'ports': []}
+        for lb in lb_db['loadbalancers']:
+            lb_port_id = lb['vip_port_id']
+            lb_subnet_id = lb['vip_subnet_id']
+            for subnet in core_db['subnets']:
+                if subnet['id'] == lb_subnet_id:
+                    updated_db['subnets'].append(subnet)
+            for port in core_db['ports']:
+                if port['id'] == lb_port_id:
+                    updated_db['ports'].append(port)
+        lb_db.update(updated_db)
+        return lb_db
+
+    def _to_api_dict(self, objs):
+        ret_list = []
+        for obj in objs:
+            ret_list.append(obj.to_api_dict())
+        return ret_list
+
+    def _get_core_context(self, context, tenant_id):
+        filters = {'tenant_id': [tenant_id]}
+        core_context_dict = common.get_core_context(context,
+                                                    filters,
+                                                    self._conf.host)
+        del core_context_dict['routers']
+        return core_context_dict
+
+    def _get_lb_context(self, context, filters):
```

Comment:
same with the [previous](https://github.com/oneconvergence/group-based-policy/pull/358#discussion_r65623136) comment